moto/rds/models.py

```diff
@@ -1485,8 +1492,12 @@ def authorize_security_group(self, security_group_name, cidr_ip):
         security_group.authorize_cidr(cidr_ip)
         return security_group
 
-    def create_subnet_group(self, subnet_name, description, subnets, tags):
-        subnet_group = SubnetGroup(subnet_name, description, subnets, tags)
+    def create_subnet_group(
+        self, subnet_name, description, subnets, tags, region, account_id
```

Comment:
The region and account are already available in the backend, so we don't have to pass them explicitly.
`self.region_name` and `self.account_id` out the top of my head